go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/sourcenetwork/defradb
 
-go 1.22
-
-toolchain go1.22.0
```

Comment:
thought:  I'm pretty sure the combination of our go version and toolchain version is the same as just setting the go version to  `go 1.22.0`. 

I'm happy to revert this if anyone can demonstrate the need to keep the toolchain directive.